erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```diff
@@ -515,10 +514,6 @@ def get_work_order_items(self, for_raw_material_request=0):
 
 		for table in [self.items, self.supplied_items]:
 			for row in table:
-				bom = get_default_bom_item(row.item_code)
-				if not bom:
-					continue
-				
 				stock_qty = row.consumed_qty if row.doctype == 'Purchase Receipt Item Supplied' else row.stock_qty
 				pending_qty = stock_qty
 				if not for_raw_material_request:
```

Comment:
@MyuddinKhatri  this condition will always be false. you are not sending any args from the calling function